assets/js/linear_algebra/dot_product_symmetric.js

```diff
@@ -168,36 +127,77 @@ function plot(scatter, axis, tt){
                   drag_start_fn=drag_start,
                   drag_end_fn=drag_end);
 
-  let texts_for_table = texts_to_show(u, v),
-      uTv_text_table = texts_for_table[0],
-      vTu_text_table = texts_for_table[1];
+  
+  let [lines_vector_u, texts_vector_u] = lib.text_matrix_to_list(
+          [[u.coord.x.toFixed(2), u.coord.y.toFixed(2), u.coord.z.toFixed(2)]],
+          [last_col_coord, start_coord_y],
+          14, [['xu', 'yu', 'zu']], 5
+          ),
+      [lines_vector_v, texts_vector_v] = lib.text_matrix_to_list(
+          [[v.coord.x.toFixed(2), v.coord.y.toFixed(2), v.coord.z.toFixed(2)]],
+          [last_col_coord, start_coord_y],
+          14, [['xv', 'yv', 'zv']], 15
+          ),
+      [lines_uT, texts_uT] = lib.text_matrix_to_list(
+          [[u.coord.x.toFixed(2)], [u.coord.y.toFixed(2)], [u.coord.z.toFixed(2)]],
+          [start_coord_x, last_row_coord],
+          14, [['xu'], ['yu'], ['zu']], 5
+          ),
+      [lines_vT, texts_vT] = lib.text_matrix_to_list(
+          [[v.coord.x.toFixed(2)], [v.coord.y.toFixed(2)], [v.coord.z.toFixed(2)]],
+          [start_coord_x, last_row_coord],
+          14, [['xv'], ['yv'], ['zv']], 15
+          );
```

Comment:
looks nice.
minor comment: instead of naming `lines_vector_u`, we should just do `lines_u`. same for `v`.

Reply:
done